bears/general/AnnotationBear.py

```diff
@@ -199,10 +200,13 @@ def get_multiline(file,
             A SourceRange object holding the range of the multi-line annotation
             and the end_position of the annotation as an integer.
         """
-        end_start = text.find(annotation_end,
-                              position + 1)
-        end_end = end_start + len(annotation_end) - 1
-        if end_start == -1:
+        try:
+            end_match = next(unescaped_search_for(annotation_end,
+                                                  text[position + 1:]))
+        except StopIteration:
+            end_match = None
+        end_end = position + end_match.span()[1] if end_match else -1
```

Comment:
It would be easier to just put the end_end calculation inside the try/cath. Why do another if?